docs/node/run-your-node/prerequisites/cloud-providers.md

```diff
@@ -0,0 +1,31 @@
+# Cloud Providers
+
+Before committing to a service be sure to verify the processor compatibility and enquire with the provider about the status of Intel SGX support. Intel maintains a comprehensive list of processors that support Intel SGX:
+
+ * https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions-processors.html
```

Comment:
Please also add this reference under the CPU section in https://deploy-preview-958--oasisprotocol-docs.netlify.app/node/run-your-node/prerequisites/hardware-recommendations#suggested-minimum-configurations